rfcs/105-trim-service-pair-prefix.md

```diff
@@ -5,6 +5,9 @@
 
```

Comment:
Maybe we need to create a tracking issue for this RFC?

Reply:
Or link implementation PR as the tracking issue?

Reply:
This GSP deprecated our public APIs.

I think we need to track the progress of service implementations so that we can decide when we finally remove the deprecated API.

For the other simple / small changes, It's OK to use the implementation PR as a tracking issue here.

Reply:
I think it doesn't deprecate public APIs?

Reply:
> I think it doesn't deprecate public APIs?

`GetObjectMetadata` has been deprecated by `GetObjectSystemMetadata` per https://github.com/beyondstorage/specs/blob/master/rfcs/117-rename-service-to-system-as-the-opposite-to-global.md#compatibility

Reply:
Do you mean GSP-117? If so, it already has a tracking issue. And this comment is on GSP-105. 👀

Reply:
Ooops, I made a mistake. (：

As you said, It's OK to link the implementation PR here.

Reply:
I'll add the implementation PR https://github.com/beyondstorage/go-storage/pull/598 as the tracking issue for GSP-105 in this PR.